app/models/katello/repository.rb

```diff
@@ -166,7 +166,7 @@ class Repository < Katello::Model
     scoped_search :on => :name, :relation => :product, :rename => :product_name
     scoped_search :on => :id, :relation => :product, :rename => :product_id, :only_explicit => true
     scoped_search :on => :label, :relation => :root, :complete_value => true, :only_explicit => true
-    scoped_search :on => :content_label, :ext_method => :search_by_content_label
+    scoped_search :on => :content_label, :ext_method => :search_by_content_label, :default_operator => :like
```

Comment:
question: is https://www.rubydoc.info/gems/scoped_search/ScopedSearch/Definition/Field#default_operator-instance_method the right place to look, to understand this does?

Reply:
ah, TY!

will give this a shot shortly when my EL9 dev environment finishes provisioning